src/lib/config.js

```diff
@@ -167,6 +169,11 @@ function getLocalConfig () {
   const expiry = {}
   expiry.minMessageWindow =
     +Config.getEnv(envPrefix, 'MIN_MESSAGE_WINDOW') || DEFAULT_MIN_MESSAGE_WINDOW
+
+  if (expiry.minMessageWindow < 1) {
+    log.warn('CONNECTOR_MIN_MESSAGE_WINDOW is less than the recommended value. Set it to a greater value to avoid lossing funds.')
```

Comment:
It would be good to explain what the problem with a low value here would be, or point to somewhere with more of an explanation. I might say something like `CONNECTOR_MIN_MESSAGE_WINDOW is less than the recommended value of 1 second. Short message windows increase the likelihood that the connector will be unable to fulfill incoming transfers before their expiries, resulting in a loss of funds. For more information on this risk, see the Interledger whitepaper`